config/routes.rb

```diff
@@ -2,4 +2,7 @@
   root 'homes#index'
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :parks do
+    resources :rides
+  end
```

Comment:
I will allow this to be merged in, but you NEED to use the `only:` syntax when you can!!!